packages/block-editor/src/components/block-variation-transforms/README.md

```diff
@@ -0,0 +1,54 @@
+# Block Variation Transforms
+
+This component allows to display the selected block's variations which have the `transform` option set in `scope` property and to choose one of them.
+
+By selecting such a variation an update to the selected block's attributes happen, based on the variation's attributes.
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders the selected block's variations which have the `transform` option set in `scope` property.
+
+```jsx
+import { useSelect } from '@wordpress/data';
+import {
+	__experimentalBlockVariationTransforms as BlockVariationTransforms,
+} from '@wordpress/block-editor';
+
+const MyBlockVariationTransforms = () => {
+	const { selectedBlockClientId } = useSelect(
+		( select ) => {
+			const { getSelectedBlockClientId } = select(
+				'core/block-editor'
+			);
+			return {
+				selectedBlockClientId: getSelectedBlockClientId(),
+			};
+		}
+	);
+
+	return (
+		<BlockVariationTransforms
+			selectedBlockClientId={ selectedBlockClientId }
```

Comment:
I know this API is still experimental, but technically speaking, it doesn't have to be a block that is currently selected so maybe we should update this prop to:
```suggestion
			blockClientId={ selectedBlockClientId }
```

What do you think?

Reply:
It makes sense, yes.